packages/components/organisms/f-checkout/src/tenants/en-NZ.js

```diff
@@ -1,4 +1,223 @@
+            },
+
+            DuplicateOrder: {
+                title: 'This order has not been placed',
+                message: 'It’s the same as the one you made a moment ago and we want to check it’s not a mistake. Wait 15 mins or change an item in your order',
+                buttonText: 'Close',
+                buttonTextPrimary: 'View my orders'
+            }
+        }
+    },
+
+    warningMessages: {
+        preOrder: {
+            title: 'Please note, this is a preorder',
+            body: 'Please check the day and time above'
+        }
+    },
+
+    buttonText: 'Go to payment',
+
+    userNote: {
+        delivery: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn\'t work). Do not include details about any allergies here.',
+            placeholder: 'e.g. the doorbell doesn’t work. Do not include details about any allergies here. \n\nWe\'re working with restaurants to cut waste. Please don\'t ask for plastic cutlery.'
+        },
+        collection: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know. Do not include details about any allergies here.',
+            placeholder: 'Do not include details about any allergies here. \n\nWe\'re working with restaurants to cut waste. Please don\'t ask for plastic cutlery.'
+        },
+        dinein: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know. Do not include details about any allergies here.',
+            placeholder: 'e.g. please bring the starters and mains at the same time. \n\nDo not include details about any allergies here.'
+        }
+
+    },
+
+    guest: {
+        firstName: 'First name',
+        lastName: 'Last name',
+        email: 'Email'
+    },
+
+    checkoutHeader: {
+        user: {
+            title: '{name}, confirm your details',
+            switchUser: 'Not {name}? Click here'
+        },
+        guest: {
+            loginTitle: 'How do you want to continue?',
+            loginButton: 'Log in or sign up',
+            option: 'or',
+            guestTitle: 'Checkout as a guest',
+            confirmation: 'Please confirm your {serviceType} details'
+        }
+    },
+
+    serviceTypes: {
+        collection: 'Collection',
+        delivery: 'Delivery'
+    },
+
+    checkoutTermsAndConditions: 'By placing an order you agree to our {termsAndConditions}. Please read our {privacyPolicy} and {cookiePolicy}.',
+
+    termsAndConditionsLinkText: 'Terms and Conditions',
+    termsAndConditionsLinkUrl: 'https://www.menulog.com.au/info/privacy-policy',
+
+    privacyPolicyLinkText: 'Privacy Policy',
+    privacyPolicyLinkUrl: 'https://www.menulog.com.au/info/privacy-policy',
+
+    cookiePolicyLinkText: 'Cookie Policy',
+    cookiePolicyLinkUrl: 'https://www.menulog.com.au/info/privacy-policy#cookies_policy',
```

Comment:
Do these links need to go to menulog.com.nz as this is for the NZ tenant?

Reply:
I need a lay down. No idea how I didn't spot that too. Updated now 🙂